spec/index.bs

```diff
@@ -884,20 +884,90 @@ The [=subresources=] of textures included in the views provided to
 {{GPURenderPassColorAttachment/resolveTarget|GPURenderPassColorAttachment.resolveTarget}}
 are considered to be used as [=internal usage/attachment=] for the [=usage scope=] of this render pass.
 
-The <dfn dfn>physical size</dfn> of a [=texture subresource=] is the dimension of the
-[=texture subresource=] in texels that includes the possible extra paddings
-to form complete [=texel blocks=] in the [=subresource=].
+The <dfn dfn>logical miplevel-specific texture extent</dfn> of a [=texture=] is the size of the
+[=texture=] in texels at a specific miplevel. It is calculated by this procedure:
 
-  - For pixel-based {{GPUTextureFormat|GPUTextureFormats}}, the [=physical size=] is always equal to the size of the [=texture subresource=]
-    used in the sampling hardware.
-  - [=Textures=] in block-based compressed {{GPUTextureFormat|GPUTextureFormats}} always have a [=mipmap level=] 0 whose {{GPUTexture/[[descriptor]]}}.{{GPUTextureDescriptor/size}}
-    is a multiple of the [=texel block size=], but the lower mipmap levels might not be
-    multiples of the [=texel block size=] and can have paddings.
+<div algorithm>
+    <dfn abstract-op>Logical miplevel-specific texture extent</dfn>(descriptor, mipLevel)
 
-<div class="example">
-Considering a {{GPUTexture}} in BC format whose {{GPUTexture/[[descriptor]]}}.{{GPUTextureDescriptor/size}} is {60, 60, 1}, when sampling
-the {{GPUTexture}} at [=mipmap level=] 2, the sampling hardware uses {15, 15, 1} as the size of the [=texture subresource=],
-while its [=physical size=] is {16, 16, 1} as the block-compression algorithm can only operate on 4x4 [=texel blocks=].
+    **Arguments:**
+        - {{GPUTextureDescriptor}} |descriptor|
+        - {{GPUSize32}} |mipLevel|
```

Comment:
nit

```suggestion

    - {{GPUTextureDescriptor}} |descriptor|
    - {{GPUSize32}} |mipLevel|
```

to make the formatting match the way it's parsed (as separate paragraphs)